test/jasmine/tests/annotations_test.js

```diff
@@ -1127,7 +1132,7 @@ describe('annotation effects', function() {
         }
 
         makePlot([
-            {x: 50, y: 50, text: 'hi', width: 50, ax: 0, ay: -40},
+            {x: 50, y: 50, text: 'hi', width: 50, ax: 0, ay: -40, xshift: -50, yshift: 50},
```

Comment:
yes, all the other tests above this in `describe('annotation effects'`, to which I added `xshift: 5, yshift: 5`, are editable and failed before I fixed dragging with shifts.